packages/apm/src/integrations/tracing.ts

```diff
@@ -789,6 +792,9 @@ export class Tracing implements Integration {
       const hub = _getCurrentHub();
       if (hub) {
         const span = activeTransaction.startChild(spanContext);
+        if (options && options.parentSpanId) {
+          span.parentSpanId = options.parentSpanId;
+        }
```

Comment:
This shouldn't be necessary you can set `parentSpanId` now in the SpanContext. 
Also, I can't fine where we ever set this?

Reply:
This is old code from an experiment previously, will remove